jax/interpreters/batching.py

```diff
@@ -35,28 +35,27 @@
 map = safe_map
 
 
```

Comment:
I'm still fairly vague about `batch` and `batch_transform`'s contracts are. A comment (types, basically) would be helpful, but I guess they are similar to other transforms, in which case a comment on any of them would be good.

Reply:
Yes, you're right. I am eager to get this in, but I've put that on my todo list.